doc/tigrc.5.adoc

```diff
@@ -803,7 +808,7 @@ View manipulation
 |quit                    |Close all views and quit
 |=============================================================================
 
-View specific actions
+View-specific actions
```

Comment:
Do you mind to also update the `REQ_GROUP("View specific actions")` in [`include/tig/request.h`](https://github.com/jonas/tig/blob/2f55f1dd0635c99ebc9d5640b149239e52e0c26d/include/tig/request.h#L42)?

There's a `make update-docs` target which extracts documentation from the code.